modules/ducktests/tests/ignitetest/services/utils/templates/discovery_macro.j2

```diff
@@ -39,6 +39,7 @@
     <bean class="org.apache.ignite.spi.discovery.zk.ZookeeperDiscoverySpi">
         <property name="zkConnectionString" value="{{ spi.connection_string }}"/>
         <property name="zkRootPath" value="{{ spi.root_path }}"/>
+        <property name="sessionTimeout" value="{{ spi.session_timeout }}"/>
```

Comment:
What profit do we have here comparing to zookeeper server min_session_timeout setup?

Reply:
The documentation [1] says: 
"The following parameters are required (other parameters are optional):

zkConnectionString - keeps the list of addresses of ZooKeeper servers.

sessionTimeout - specifies the time after which an Ignite node is considered disconnected if it doesn’t react to events exchanged via Discovery SPI.​"

[1] https://ignite.apache.org/docs/latest/clustering/zookeeper-discovery